### PR TITLE
Top Crypto Prices - Finished the app

### PR DESCRIPTION
**Pull request by playak**
Does all I wanted it to do now, including quite some config options.

### CODE REVIEW

File: apps/topcryptoprices/top_crypto_prices.star [hunk 1]
```diff
@@ -0,0 +1,331 @@
...
+load("render.star", "render")
+load("http.star", "http")
+load("encoding/base64.star", "base64")
+load("encoding/json.star", "json")
+load("cache.star", "cache")
+load("math.star", "math")
+load("schema.star", "schema")
+
+# INIT
+credits = ["Data:Coingecko", "Code:Playak"]  # Credits required for using free Coingecko API
```
Comment (matslina) [inline apps/topcryptoprices/top_crypto_prices.star:32]: Giving credit in the app description should be sufficient to meet the API terms, and could perhaps make for a better user experience?

File: apps/topcryptoprices/top_crypto_prices.star [hunk 2]
```diff
@@ -0,0 +1,331 @@
...
+Author: playak
+
+Version: 1.1
+Date: 2022-11-09
+Comments: Written to display the price info of top cryptocurrencies on a Tidbyt screen. We manually exclude stable coins and wrapped coins (so we only need one API call).
+Copyright: © 2022 Jeroen Houttuin, Playak - jeroen@playak.com - https://playak.com
+"""
+
+# CONFIG
+TTL = 60  # time to live for the API cache. Be nice to the free Coingecko API provider!
```
Comment (matslina) [inline apps/topcryptoprices/top_crypto_prices.star:14]: This is probably good enough, but should mention that there are other apps using the free API as well, so they'll all be sharing the same rate limit (although I'm not sure how Coingecko actually implements that, if at all). If we run into problems we can move this app over to Tidbyt's paid API key.

**Review by matslina — CHANGES_REQUESTED** [review]
Thanks for the PR! This looks pretty good. Added a few thoughts in inline comments.

The main blocker for merging right now is just the `apps/topcryptos` directory that looks like a leftover from early development? Also need to remove the references to that app in `apps/apps.go`.

**Review by matslina — APPROVED** [review]

File: apps/topcryptoprices/top_crypto_prices.star (hunk 2 above)
Comment (matslina) [reply apps/topcryptoprices/top_crypto_prices.star:14]: Looks like you got it. I'll merge this and try to get it published later today.

File: apps/topcryptoprices/top_crypto_prices.star (hunk 1 above)
Comment (playak) [reply apps/topcryptoprices/top_crypto_prices.star:32]: Good idea. I made that configurable now, as an extra option. Default is to not show the credits on the Tidbyt.

**Pull request merged**